org.cockpit_project.CockpitClient.yml

```diff
@@ -0,0 +1,38 @@
+app-id: org.cockpit_project.CockpitClient
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+default-branch: stable
```

Comment:
```suggestion
```

Reply:
This is here because this file is generated out of upstream git and we sometimes want to write `default-branch: devel` here (in case it's not for a released tarball).  We could maybe change that around to use a commandline argument to `flatpak-builder` but this seemed a lot easier.